spec/models/miq_provision_request_spec.rb

```diff
@@ -2,19 +2,31 @@
 
 describe MiqProvisionRequest do
   it ".request_task_class_from" do
```

Comment:
Should this be changed to a context ".request_task_class_from" with an it for each provider type?